### PR TITLE
WIP/NO-MERGE feat: Bump path-to-regexp

### PR DESCRIPTION
**Pull request by astuyve**
### What does this PR do?
attempting #2729 to help alleviate https://github.com/DataDog/serverless-plugin-datadog/issues/321

### Motivation
[
](https://github.com/DataDog/serverless-plugin-datadog/issues/321)
### Plugin Checklist

- [ ] Unit tests.
- [ ] TypeScript [definitions][1].
- [ ] TypeScript [tests][2].
- [ ] API [documentation][3].
- [ ] CircleCI [jobs/workflows][4].
- [ ] Plugin is [exported][5].

[1]: https://github.com/DataDog/dd-trace-js/blob/master/index.d.ts
[2]: https://github.com/DataDog/dd-trace-js/blob/master/docs/test.ts
[3]: https://github.com/DataDog/documentation/blob/master/content/en/tracing/trace_collection/library_config/nodejs.md
[4]: https://github.com/DataDog/dd-trace-js/blob/master/.circleci/config.yml
[5]: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/plugins/index.js

### CODE REVIEW

**Review by Qard — APPROVED** [review]

**Comment by astuyve** [comment]
As mind-boggling as this is, the test failure is correct and only occurs on this branch. I need to spend some time investigating.

DO NOT MERGE

**Comment by astuyve** [comment]
As mind-boggling as this is, the test failure is correct and only occurs on this branch. I have reproduced this locally.

I need to spend some time investigating before merging

**Comment by tlhunter** [comment]
I'll close this for now since it seems like maybe a temporary issue. Feel free to open back up if I'm wrong!

**Pull request closed without merging**
